PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -488,6 +499,11 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
             addToProcessAndTask('pat'+metType+'Met'+postfix,  getattr(process,'patPFMet' ).clone(), process, task)
             getattr(process, "patPFMet"+postfix).metSource = cms.InputTag("pfMet"+postfix)
             getattr(process, "patPFMet"+postfix).srcPFCands = self._parameters["pfCandCollection"].value
+        if metType == "PF":
+            getattr(process, "patPFMet"+postfix).srcLeptons = cms.VInputTag(cms.InputTag("pfEGammaToCandidate","electrons") if not self._parameters["onMiniAOD"].value else self._parameters["electronCollection"].value,
```

Comment:
here and a bit above, there is some lack in consistency between using ```'pat'+metType+'Met'+postfix``` and ```"patPFMet"+postfix```. The coding pattern is suggestive that it's supposed to be easy to change ```metType``` value; perhaps define ```_myPatMet = 'pat'+metType+'Met'+postfix``` on L495 and use it everywhere.